clang/test/CodeGenHLSL/builtins/ldexp.hlsl

```diff
@@ -0,0 +1,49 @@
+// RUN: %clang_cc1 -finclude-default-header -x hlsl -triple dxil-pc-shadermodel6.3-library %s -fnative-half-type -emit-llvm -disable-llvm-passes -o - | FileCheck %s
```

Comment:
A lot of the builtins that support half and float have tests with native half and without native half.
See: https://github.com/llvm/llvm-project/blob/main/clang/test/CodeGenHLSL/builtins/cos.hlsl
Should we do that here?

Reply:
Yea, to be consistent if nothing else.

Reply:
I was hoping we would only do those for intrinsics that require clang builtins since we define most of them as `void(...)`.   for anything thats pure HLSL like this one we have extensive testing of native half.